apps/mobile/src/features/send/send-sheet-content.tsx

```diff
@@ -0,0 +1,21 @@
+import { useSendContext } from './send-context';
+import { SelectAccount } from './send-steps/select-account';
+import { SelectAsset } from './send-steps/select-asset';
+import { SendForm } from './send-steps/send-form';
+
+export function SendSheetContent() {
+  const { step } = useSendContext();
+
+  switch (step) {
```

Comment:
Could we look into using the router here? Using a context, we should be able to do this without passing state between steps.

Reply:
When using sheets, I didn't see anyone suggesting using the router for this ...this was the recommended approach. @edgarkhanzadian am I wrong here? Happy to change, but not sure why this is a bad approach? I thought we were trying to avoid route state with the forms? Didn't we go that direction in the extension and wanted to avoid it this time?

Reply:
@kyranjamie I think I misunderstood your comment here? You mean just for the `account` and `asset`, not for the form state? Let me look bc this is what I thought was suggested to do, but maybe I can use a nested Stack.

Reply:
>  I thought we were trying to avoid route state with the forms?

In the extension there was a gross overuse of form state. Passing everything from tx details, and pre-formatted sums of money etc. Really bad. IMO routes should be used for state that represents _a location within the app_. 

Maybe there are complexities with the sheet I'm unclear on, that mean we have to do it this way. 

But, I would typically argue this is a "bad approach" because it reimplements a much less capable router. One example: What if we need to navigate straight into the form, because from where the user initiates it the send, `account` and `asset` are already known. That's easy if we know the routing scheme is `/send/[fingerprint][accountId]/stx` or something like that. @edgarkhanzadian is already using the router to pass wallet/account data in some places.